<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="boardWidget" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>450</width>
     <height>450</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="numbersWidget" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>251</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="eraseButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="undoButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>20</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="notesButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>20</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="gameOverDialog" native="true">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>431</y>
     <width>191</width>
     <height>170</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="winGame">
   <property name="geometry">
    <rect>
     <x>403</x>
     <y>431</y>
     <width>95</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Win the game!</string>
   </property>
  </widget>
  <widget class="QLabel" name="mistakesLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>143</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mistakes: 0/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>410</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>121</width>
     <height>60</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
